common/helpers/date.js

```diff
@@ -72,4 +68,28 @@ module.exports = {
       format(dateTo, DATE_FORMATS.URL_PARAM),
     ]
   },
+
+  isCurrentWeek(date) {
```

Comment:
I'm not sure if it is clearer to have slightly duplicated code, or have to have a function that adds `-1`, `0` and `1` to the current week for last, current and next week calculations.

Reply:
I think date-fns already has this method: [`isThisWeek`](https://date-fns.org/v2.16.1/docs/isThisWeek)

Reply:
Related to the comments above, yes it does have `isThisWeek`, but it doesn't have `isPreviousWeek` or `isNextWeek` - thus my question about how best to code this up so it's easier to understand.

As per above, I'll just remove it for the time being.